### ISSUE TITLE
worse scores

### ISSUE THREAD

**Issue by bolin12** [PRIS-CV/PMG-Progressive-Multi-Granularity-Training#6]
dear author：
    I got worse scores in my machine with backbone of pretrained resnet50 on CUB(84.06), 20 epoch, another is even worse(81.42), on the final stage, training loss is near to zero and training acc is 100.

   can you solve my problem? thanks

**Comment by RuoyiDu** [comment]
> dear author：
> I got worse scores in my machine with backbone of pretrained resnet50 on CUB(84.06), 20 epoch, another is even worse(81.42), on the final stage, training loss is near to zero and training acc is 100.
> can you solve my problem? thanks
Have you ever changed any settings? And please make sure your torch version is higher than 1.3.0

**Comment by bolin12** [comment]
pytorch 1.4 and torchvision 0.5
